game/app/singleton/player_manager.gd

```diff
@@ -0,0 +1,5 @@
+extends Node
+
+
+func get_current_player() -> String:
```

Comment:
Why move that here, if it still has no functionality?

Reply:
a. why should the GameManager handle the players, even if it is not really functional yet?
b. de-clutters the GameManager